app/views/layouts/member_facing.slim

```diff
@@ -20,6 +18,9 @@ html lang="#{@page.language.try(:code).try(:downcase)}"
     = render partial: 'shared/optimizely_snippet' unless @page.optimizely_disabled?
     = javascript_include_tag 'globals'
     = render partial: 'shared/js_locale'
+    
+    = render 'layouts/mixpanel' if Settings.mixpanel_token
```

Comment:
This isn't related to your change but rather to the old code - how come the mixpanel markup is under layouts rather than a partial? Otherwise good to go IMO, granted that it works.

Reply:
yeah, not really sure why! I'll be taking mixpanel out in the next few days once @joeconey is done using it.